ci/Jenkinsfile.e2e-nightly

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env groovy
+library 'status-jenkins-lib@v1.7.9'
+
+pipeline {
+  agent { label 'linux || macos || windows' }
```

Comment:
Not sure if that makes sense. We have a lot of linux hosts, and not many others, might be bad for the queue.